GoogleApisClient.sln

```diff
@@ -48,6 +48,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoogleApis.WP81", "Src\Goog
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoogleApis.Auth.WP81", "Src\GoogleApis.Auth.WP81\GoogleApis.Auth.WP81.csproj", "{90660B22-B933-44A6-B34D-E4F4352D4434}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoogleApis.Auth.DotNet4.Tests", "Src\GoogleApis.Auth.DotNet4.Tests\GoogleApis.Auth.DotNet4.Tests.csproj", "{5A731729-8E99-4930-AE77-FA3524673DA3}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Google Compute Sample", "..\..\..\Documents\Visual Studio 2013\Projects\Google Compute Sample\Google Compute Sample\Google Compute Sample.csproj", "{8A409FA2-6788-468B-A156-BA91173116DA}"

```

Comment:
Please revert.
I'll be happy if we will have a sample in the samples repository.
https://github.com/google/google-api-dotnet-client-samples